docs/manual/related-projects.md

```diff
@@ -205,3 +226,39 @@ See
 [bmap](https://source.tizen.org/documentation/reference/bmaptool/introduction).
 A tool for optimized copying of disk images. Intended for offline use,
 so not directly comparable.
+
+## Git
+
+Although OSTree has been called "Git for Binaries", and the two share the idea
+of a hashed content store, the implementation details are quite different.
+OSTree supports extended attributes and uses SHA256 instead of Git's SHA1. It
+"checks out" files via hardlinks, rather than copying, and thus requires the
+files to be immutable. At the moment, OSTree commits may have at most one
+parent, as opposed to Git which allows an arbitrary number. Git uses a
+smart-delta protocol for updates, while OSTree uses (slow) HTTP requests or
+static deltas.
+
+## Conda
+
+Conda is an "OS-agnostic, system-level binary package manager and ecosystem";
```

Comment:
Interesting, I hadn't really looked at this before.  Can you hyperlink it here too?